com.unity.multiplayer.mlapi/Runtime/Core/NetworkingManager.cs

```diff
@@ -383,6 +383,15 @@ private void Init(bool server)
                 return;
             }
 
+            rpcQueueContainer = new RpcQueueContainer(false);
```

Comment:
Since we've learned that there is some subtle, serious sensitivity to the RPC queue init, I'd like a comment saying something like "This must happen in Init(), not Awake() / OnEnable() because...."

Reply:
I would also argue that RpcQueueContainer should be marked as internal thus not allowing it to leak out to the SDK because I really can't see a need for someone to create a queue themselves and would also prevent them trying to do something weird with it in user code

Reply:
+1 on both points!